### DISCUSSION TITLE
Small typo on ordered_values example.

### DISCUSSION THREAD

**Discussion by siloegarcez** [sqlalchemy/sqlalchemy#10982] [Ideas]
Just a small typo on the ordered_values example in the docs 😊.

https://github.com/sqlalchemy/sqlalchemy/blob/81682347569d83ed5f23eaf51b29786ad21a29f8/lib/sqlalchemy/sql/dml.py#L1554

Should be:

E.g.::

            stmt = table.update().ordered_values(
                ("name", "ed"), ("ident", "foo")
            )

**Comment by CaselIT** [comment]
Feel free to just send a PR for such doc changes

**Reply by siloegarcez** [reply]
Ok, will send a PR!
